### DISCUSSION TITLE
Welcome to unitycatalog-ui Discussions!

### DISCUSSION THREAD

**Discussion by yc-shawn** [unitycatalog/unitycatalog-ui#44] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by everettVT** [comment]
Hi all, 

I'm Everett and I'm excited for uc oss to help us simplify data accessibility. I am eagerly integrating UC as my primary data control plane and am excited by the ui support. I'm still new to contributing, but I would love to help where I can.
